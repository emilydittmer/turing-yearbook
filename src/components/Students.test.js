import React from 'react';
import { shallow, mount } from 'enzyme';
import Student from './Students';

describe('Students', () => {

  let students;
  let wrapper;

  beforeEach(() => {
    students = [
      {
      id: 1, 
      name: 'Emily',
      quote: 'Quote',
      superlative: 'Superlative',
      photo: 'https://placekitten.com/200/300'
      },
      {
      id: 2, 
      name: 'Jasper',
      quote: 'Quote',
      superlative: 'Superlative',
      photo: 'https://placekitten.com/200/300'
      }
    ]

    wrapper = shallow(<Student 
        students={students}
      />)
  })
  it('should match the snapshot will all data passed in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  })
})