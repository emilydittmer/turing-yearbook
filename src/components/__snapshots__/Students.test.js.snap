// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Students should match the snapshot will all data passed in correctly 1`] = `
<section
  className="students"
>
  <Person
    id={1}
    key="1"
    name="Emily"
    photo="https://placekitten.com/200/300"
    quote="Quote"
    superlative="Superlative"
  />
  <Person
    id={2}
    key="2"
    name="Jasper"
    photo="https://placekitten.com/200/300"
    quote="Quote"
    superlative="Superlative"
  />
</section>
`;
